views/includes/entity.html

```diff
@@ -71,6 +71,42 @@
   %}
 {% endif %}
 
+{% if pageParams.isRegistration %}
```

Comment:
Wondering why all this new template code was added for the registration journey as defect only talks about removing the change link for the update journey?

Reply:
That is actually the existing code, we have just replaced it with a variable so we can cover the difference with update as well.

Reply:
👍 thanks.